.github/workflows/publish_conda.yaml

```diff
@@ -0,0 +1,24 @@
+name: Publish the CLI on Conda
+
+on:
+  push:
+    tags:
+      - "*"
+jobs:
+  upload_to_conda:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Anaconda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-activate-base: true
+          activate-environment: ""
+      - name: Conda Upload
+        shell: bash -l {0}
+        run: |
+          conda install conda-build
+          conda install anaconda-client
+          conda install conda-verify
+          conda build --output-folder ./conda-out/ ./fidesctl/conda/ -c plotly -c conda-forge
+          anaconda -t "${{ secrets.CONDA_TOKEN }}" upload -u ethyca ./conda-out/noarch/fidesctl*.bz2
```

Comment:
Here is the run verifying this works now:
https://github.com/ethyca/fides/actions/runs/1501640432

Reply:
thank you!